### ISSUE TITLE
Help wanted: Zoom ISV partner application

### ISSUE THREAD

**Issue by Jericho** [Jericho/ZoomNet#160]
I recently submitted an application to join Zoom's ISV partner program which would grant me access to additional API functionality and ultimately allow me to add additional functionality to ZoomNet. 

As part of this application process they asked about my revenue model, how my project benefits Zoom (for instance, do I resell their product), how my project benefits their customers, etc. I disclosed that I run an open source project and that I don't derive any revenue from it, I don't resell their product and I am not involved in anybody's decision to sign up for Zoom's service. I do believe that the availability of an open-source library for the .NET community can certainly influence the buying decision of some of their prospective customers but I can't quantify it.

Anyway, long story short: I am concerned that the fact that I don't DIRECTLY influence customers and the fact that I don't bring in additional revenue to Zoom might be viewed detrimentally and ultimately result in my application being rejected. My fingers are crossed and I hope that I'm wrong about this but, we'll see. I'll make sure to keep everyone in the loop if/when I get news.

In the mean time, does anybody know someone at Zoom that can influence this decision? Can you help me make my case?

**Comment by cdmdotnet** [comment]
I don't know anyone, but good luck and keep us updated.
It might pay to say that revenue for Zoom is indirect. Indirect in that you don't generate revenue for them, but enable other ISV's to quickly create solutions that will generate revenue for Zoom.

**Comment by Sanx78** [comment]
I realise this is a rather old post, but I've contacted our Zoom account manager and customer success manager to ask for their assistance.

**Comment by Jericho** [comment]
@Sanx78 Thank you for doing that, much appreciated.

**Comment by Jericho** [comment]
It's been almost 3 and a half years since I wrote this plea for help and I am pleased to announce that there's been some progress. Somebody at Zoom noticed the work I have been doing for the community and nominated me for the MVP program... and my application was recently approved! As part of the program I was granted access to additional functionality such as Events, Phone, Rooms, Contact Center, etc.

This means that I'll be able to add new features to the ZoomNet library. You can expect these new features to roll out over the next several months.
